tsfile/src/main/java/org/apache/iotdb/tsfile/common/conf/TSFileConfig.java

```diff
@@ -75,7 +75,7 @@
   /** The maximum number of data points in a page, default value is 1024 * 1024. */
   private int maxNumberOfPointsInPage = 1024 * 1024;
   /** The maximum degree of a metadataIndex node, default value is 1024 */
-  private int maxDegreeOfIndexNode = 256;
+  private int maxDegreeOfIndexNode = 1024;
```

Comment:
hi , maxDegreeOfIndexNode should be Consistent as config item.  you should alter this config too. 
https://github.com/apache/iotdb/blob/4f4ee906c6a10915d86ce900ede7174ab161fbfa/server/src/assembly/resources/conf/iotdb-engine.properties#L570

Reply:
OK, Thanks for your careful review~